src/Standards/PEAR/Tests/Functions/FunctionCallSignatureUnitTest.php

```diff
@@ -135,7 +129,6 @@ public function getErrorList($testFile='FunctionCallSignatureUnitTest.inc')
             567 => 1,
             568 => 1,
             573 => 1,
-            574 => 1,
```

Comment:
All these errors no longer being thrown, gives me the impression you are _breaking_ the PEAR sniff behaviour in favour of fixing something which affects only PSR2. That doesn't seem right....

What is the justification for this ?

Reply:
I've gone back through these changes and re-assessed each in turn. Below are my findings, grouped together by class / reasoning.

<details><summary>Lines 106 & 355</summary>

There were two errors being flagged for this line:

    106 | ERROR | [x] Multi-line function call not indented correctly; expected 0 spaces but found 4 (PEAR.Functions.FunctionCallSignature.Indent)
    106 | ERROR | [x] Closing parenthesis of a multi-line function call must be on a line by itself (PEAR.Functions.FunctionCallSignature.CloseBracketLine)

The first error is actually not correct (or is misleading). The parameter is correctly indented by 4 spaces. The closing parenthesis should be indented by zero spaces, but not on this line. Fixing the second error (which is still reported after the changes in this pull request) by moving the closing parenthesis to its own line makes this clear as the indents for the parameter and closing parenthesis are then checked/reported independently.

The error message when the line is incorrectly indented (eg with 5 spaces) is misleading as it reports "expected 0 spaces but found 5" whereas it should probably report "expected 4 spaces but found 5" for the parameter (and leave the closing parenthesis to report its own indentation anomalies when it's on its own line.). I'm not trying to fix that problem here; this is an improvement that could be made in the future.

<hr></details>

<details><summary>Lines 378-380 & 386-388 (& 394-396)</summary>

This code snippet shows these errors:

    377 | ERROR | [x] Opening statement of multi-line function call not indented correctly; expected 4 spaces but found 5 (PEAR.Functions.FunctionCallSignature.OpeningIndent)
    378 | ERROR | [x] Multi-line function call not indented correctly; expected 9 spaces but found 8 (PEAR.Functions.FunctionCallSignature.Indent)
    379 | ERROR | [x] Multi-line function call not indented correctly; expected 9 spaces but found 8 (PEAR.Functions.FunctionCallSignature.Indent)
    380 | ERROR | [x] Multi-line function call not indented correctly; expected 5 spaces but found 4 (PEAR.Functions.FunctionCallSignature.Indent)

There is ambiguity in the sniff here in that it will accept any multiple of 4 spaces (including zero) on line 377. When line 377 is no longer being flagged as problematic, the following three lines are either correct (when 377 has four spaces) or off by some multiple of four (eg, when 377 has 12 leading spaces, the following three lines are each indented by 8 too few spaces).

[The coding standard does say that "Parameters need to be indented 4 spaces compared to the level of the function call"](https://pear.php.net/manual/en/standards.funcalls.php), so I can see why the following three lines are being reported as "wrong" by this sniff. However [the same coding standard says to "Use an indent of 4 spaces"](https://pear.php.net/manual/en/standards.indenting.php), which makes me think that "expected 9 spaces" and "expected 5 spaces" are in violation of the coding standard.

Lines 394-396 don't show up in the delta for this pull request an error message is still shown for these lines, but the message is changing in this pull request. Previously the sniff would suggest an indent of 13/9 spaces, now the sniff suggests an indent of 12/8 spaces.

<hr></details>

<details><summary>Line 574</summary>

This one is harder to justify. The sniff (now) accepts both 4 & 6 spaces indentation here as "correct." Should we be accepting only one of these numbers? I can't see any examples of "inline HTML" in the coding standard at https://pear.php.net/manual/en/standards.php

</details>